tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -205,14 +231,13 @@
               edpm_sshd_configure_firewall: true
               edpm_sshd_allowed_ranges: ['192.168.122.0/24']
               # SELinux module
-              edpm_selinux_mode: permissive
-              edpm_hosts_entries_undercloud_hosts_entries: []
-              # edpm_hosts_entries role
-              edpm_hosts_entries_extra_hosts_entries:
-              - 172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc
-              - 172.17.0.85 rabbitmq.openstack.svc
-              - 172.17.0.86 rabbitmq-cell1.openstack.svc
-              edpm_hosts_entries_vip_hosts_entries: []
-              hosts_entries: []
-              hosts_entry: []
+              edpm_selinux_mode: enforcing
+              plan: overcloud
     EOF
+
+- name: wait for dataplane to be ready
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc wait --for condition=Ready osdp/openstack --timeout=30m
+  when: "edpm_node_ip.startswith('192.168.122')"
```

Comment:
temporary workaround for `centos-9-standalone-to-crc-data-plane-adoption`

Reply:
do you want to add a #TODO(fao89): 
so we can revisit that then?
but what will the proper fix be? why is it temporary? i think we will only ever run that for the single node job?

Reply:
yes, I will add a TODO
I'm learning about the [networking config](https://github.com/openstack-k8s-operators/docs/blob/main/networking.md) and I noticed that with the current install_yamls way, many places expect the ip range to be under `192.168.122`, which would require changes in other test roles and on CI config. As we have a job already working, I think we can enable it, and I can work on the other job in a follow-up PR